vendor/bat-native-ads/src/bat/ads/internal/bundle.cc

```diff
@@ -188,6 +193,50 @@ std::unique_ptr<BundleState> Bundle::GenerateFromCatalog(
   return state;
 }
 
+bool Bundle::DoesOsSupportCreativeSet(
+    const CreativeSetInfo& creative_set) {
+  if (creative_set.oses.empty()) {
+    // Creative set supports all OSes
+    return true;
+  }
```

Comment:
Just want to check here - if there's a server bug where the feed for `oses` is empty _or_ if the JSON object is renamed and the parser isn't updated to catch them, then this will allow all creative sets on all OS's - is that acceptable default behaviour?
(If the feed is renamed and the set is for OSX only, then this will allow the set on Windows, iOS, Android... )

Reply:
This is use case specified at https://github.com/brave/brave-browser/issues/4761 which would hopefully get picked up when testing future change requests